### ISSUE TITLE
Amulet crash POB on import, or maybe combo of items do it, not sure it's the amu anymore

### ISSUE THREAD

**Issue by iDvoth** [PathOfBuildingCommunity/PathOfBuilding#1876]
I just updated to the lastest version today and this item crashed it everytime.

![Awakened_PoE_Trade_oW4udFqRVc](https://user-images.githubusercontent.com/65934484/104482705-3dc8f400-5595-11eb-804f-496f4de53f87.png)

This is it's copy code:
https://pastebin.com/0U8nnizP

I get this error message:
![Path_of_Building_hC1UgAXLd3](https://user-images.githubusercontent.com/65934484/104482482-fc384900-5594-11eb-874e-69a07e3cca0d.png)

And this after pressing ESC:
![Path_of_Building_QwDNGtQbBW](https://user-images.githubusercontent.com/65934484/104482575-12dea000-5595-11eb-9ba3-e84398f43e12.png)

**Comment by iDvoth** [comment]
Ok I've tested it more, the item itself doesn't seem to be the only cause. https://pastebin.com/vJzLXrf8 this is the build that has this item, and this build can't be loaded into pob anymore. In fact if I import data from my poe profile from that character it crashes with the same message as above. It crashes at the moment I click to import items. And when I tried rebuilding the build manually in pob it crashed at the point of which I added that item so though that was the problem. However if you import this item in a clean build it works just fine.
